<launch>

  <node pkg="asterx1_node"
        type="raw_receiver_node"
        name="raw_receiver_node" 
        output="screen">
    <!--<remap from="/iri_asterx1_gps/gps_meas" to="/teo/sensors/gps/gps_meas"/>
    <!--<remap from="/iri_asterx1_gps/gps_raw_data" to="/teo/sensors/gps/gps_raw_data"/>-->
  </node>

  <node pkg="rosbag"
        type="play"
        name="player" 
        output="screen"
        args="/home/ptirindelli/rosbags/teo/2016-03-03-10-31-45.bag -r 4 -l"/>

</launch>

